databases/orientdb/src/main/java/com/gentics/mesh/graphdb/cluster/OrientDBClusterManager.java

```diff
@@ -372,26 +390,94 @@ public void startServer() throws Exception {
 
 		log.info("Starting OrientDB Server");
 		server.startup(getOrientServerConfig());
```

Comment:
Are you sure? I dislike the idea of hot-patching config files on the fly. This can create confusion if someone is expecting the plugin to be used but the patching mechanism changes this internally. This is not visible to the user / dev.